lib/v2/screens/dashboard/wallet_screen.dart

```diff
@@ -72,29 +89,31 @@ class _WalletScreenState extends State<WalletScreen> with AutomaticKeepAliveClie
     );
   }
 
-  Widget buildAppBar(BuildContext context) {
+  Widget buildAppBar(BuildContext context, String account, String? nickname, String? image) {
```

Comment:
Convert this function to Widget class in a separate file please.

```
class WalletAppBar extends StatelessWidget implements PreferredSizeWidget {
  const WalletAppBar({Key? key}) : super(key: key);

  @override
  Size get preferredSize => Size.fromHeight(kToolbarHeight);

  @override
  Widget build(BuildContext context) {
    return BlocBuilder<WalletBloc, WalletState>(
      builder: (context, state) {
        return AppBar(
          actions: [
            const SizedBox(width: horizontalEdgePadding),
            IconButton(
              iconSize: 36,
              splashRadius: 26,
              onPressed: () => NavigationService.of(context).navigateTo(Routes.profile),
              icon: ProfileAvatar(
                size: 36,
                account: state.profile.account,
                nickname: state.profile.nickname,
                image: state.profile.image,
              ),
            ),
            Expanded(child: Image.asset('assets/images/seeds_symbol_forest.png', height: 56, fit: BoxFit.fitHeight)),
            IconButton(
              splashRadius: 26,
              icon: SvgPicture.asset('assets/images/wallet/app_bar/scan_qr_code_icon.svg', height: 36),
              onPressed: () => NavigationService.of(context).navigateTo(Routes.scanQRCode),
            ),
            const SizedBox(width: horizontalEdgePadding),
          ],
        );
      },
    );
  }
}
```

Reply:
@override
  Size get preferredSize => Size.fromHeight(kToolbarHeight); 

I have to try this

Reply:
It must works.
`kToolbarHeight` is a Flutter constant for the appBar height.